### PR TITLE
Additional regression test failure fixes 204 926

### PR DESCRIPTION
**Pull request by bwelch8105**
This PR contains additional updates and fixes for regression test failures.

### CODE REVIEW

File: cypress/e2e/WebInterface/Measure/CreateMeasure/CreateMeasureValidations.cy.ts [hunk 1]
```diff
@@ -1049,11 +1049,11 @@ describe('Measure Creation: CV ListQDMPositiveEncounterPerformed With MO And Str
             .then((radio) => {
                 //confirm that initial value is set to 'No'
                 cy.wrap(radio).eq(1).should('be.checked');
-                cy.contains('[class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1jaw3da"]', 'No');
+                cy.pause()
```
Comment (cvasile001) [inline cypress/e2e/WebInterface/Measure/CreateMeasure/CreateMeasureValidations.cy.ts:1052]: pause here

File: cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts [hunk 2]
```diff
@@ -407,7 +408,7 @@ describe('Validating Stratification tabs', () => {
         cy.get(MeasureGroupPage.stratAssociationOne).should('have.attr', 'placeholder', 'Select All That Apply')
         //Association -- contains these values based off score type -- score type is Continuous Variable
         cy.get(MeasureGroupPage.stratAssociationOne).click()
-        cy.get('[class="base-Popper-root MuiAutocomplete-popper MuiAutocomplete-popperDisablePortal css-1pxin5q"]').each(($ele) => {
+        cy.get('[id="association-select-1-listbox"]').each(($ele) => {
```
Comment (cvasile001) [inline cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts:411]: this is exactly what I was talking about. Thanks Brandon!!

**Review by cvasile001 — CHANGES_REQUESTED** [review]
I think we just want to remove that pause, everything else looks good. Thanks for getting in those quick refactor updates on those IDs!!

File: cypress/e2e/WebInterface/Measure/CreateMeasure/CreateMeasureValidations.cy.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/e2e/WebInterface/Measure/CreateMeasure/CreateMeasureValidations.cy.ts]: Doh!
I can't believe I left that in there! LOL
I just removed this, so it _should_ be good-to-go, now.

File: cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts (hunk 2 above)
Comment (bwelch8105) [reply cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts:411]: No problem
Apologies for not doing this the first go around.

**Review by cvasile001 — APPROVED** [review]
approved

**Pull request merged**